generators/app/templates/src/app/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+interface Props {
+  checked?: boolean;
```

Comment:
If we make `onChange` optional, does it make sense to have the `isChecked` state?
Right now the `checked` value is only being used for the initial state. That's also a problem.

Reply:
We can either remove the state and make onChange required, or keep it optional and use a useEffect to keep the state synced with the prop.

Reply:
It's true! `name`, `onChange` and `checked` must be mandatory, and local state makes no sense. 
But, `disabled` and `required` should have *false* as default so that the component is less strict.